match/lib/match/options.rb

```diff
@@ -55,6 +55,12 @@ def self.available_options
                                      description: "Only fetch existing certificates and profiles, don't generate new ones",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :disable_encryption,
+                                     env_name: "MATCH_DISABLE_ENCRYPTION",
+                                     short_option: "-d",
+                                     description: "Disable encryption, certificates including keys and profiles won't be encrypted! be sure that your git repo is safe!",
```

Comment:
In general, I'm wondering if we can show an error message if you run _match_ without encryption, but for an encrypted repo, and the other way around. How does _match_ currently behave in that situation?